Composer/packages/client/src/components/GetStarted/GetStartedTask.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { Fragment } from 'react';
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+
+import { NextSteps } from './GetStartedNextSteps';
+
+type TaskProps = {
+  step: NextSteps;
+};
+
+export const GetStartedTask: React.FC<TaskProps> = (props) => {
+  const icon = props.step.checked ? 'CompletedSolid' : props.step.required ? 'Error' : 'Completed';
+  const color = props.step.checked ? '#219653' : props.step.required ? '#ca5010' : '#0078D4';
```

Comment:
Could we pull these colors from a Fabric palette?

Reply:
yup!